docs/theory.rst

```diff
@@ -156,12 +161,12 @@ Then, defining :math:`\mathbf{x}_i` to be:
    \mathbf{x}_i = [\mathbf{y}_i \quad \mathbf{r}_{ui} \quad \mathbf{p}_{ui}]^T
 
 
-The above equations will create :math:`nm` constraint equations and the
+The above equations will create :math:`n(N-1)` constraint equations and the
 optimization problem can formally be written as:
 
 .. math::
 
-   & \underset{\mathbf{x}_i \in \mathbb{R}^{(n + p)m + q}}
+   & \underset{\mathbf{x}_i \in \mathbb{R}^{(n + q)N + r}}
```

Comment:
x_i has dimension n + q + r
it is the vector (y_1, r_u1, y_2, r_u2, ...p_u) that has dimension (n + q)N + r